### PR TITLE
[student pack] Rename button to "Claim Offer"

### PR DESCRIPTION
**Pull request by 32leaves**
fixes https://github.com/github-student-developer-pack/Gitpod/issues/12

### CODE REVIEW

**Comment by jankeromnes** [comment]
Also, for urgent fixes, in theory you should be able to merge changes without requesting a review (even though in general it’s better to do so).

**Comment by elenalape** [comment]
@jankeromnes What I meant is also to change the functionality of the button so that the students can claim offer using it.

**Comment by jankeromnes** [comment]
@elenalape Thanks for the clarification! 💯 

Sorry, I don't have access to https://github.com/github-student-developer-pack/Gitpod/issues/12, so I'm not fully aware of the context. But is the requirement to:

1. have one "Claim Offer" button in the page, or
2. that the primary Call To Action should be "Claim Offer"?

I ask, because below the plan boxes in https://www.gitpod.io/github-student-developer-pack/, there is already a "Claim Offer" button.

**Comment by elenalape** [comment]
@jankeromnes 👋 Hello! Happy to provide some clarification, let's work on this together.

From a UX perspective, is desirable that the main thing the student sees when they land on your page is a `Claim offer` button that takes the student through your offer redemption flow. I personally don't think that it is an issue if your landing page contains multiple `Claim offer` buttons.

I have added you as a collaborator in that repo where we can discuss it further in private.

File: src/pages/github-student-developer-pack.tsx [hunk 1]
```diff
@@ -248,7 +248,7 @@ const CareersPage: React.SFC<{}> = () => (
                     <UnderLine />
                     <p>With Gitpod you have no more tedious setups,<br /> you save hours of compiling code,<br /> and you can start coding from any device, immediately.</p>
                     <a href="https://gitpod.io/#https://github.com/gitpod-io/NextSimpleStarter" target="_blank">
```
Comment (elenalape) [inline src/pages/github-student-developer-pack.tsx]: The `Claim offer` should link to your offer redemption flow, not a demo.

Comment (jankeromnes) [reply src/pages/github-student-developer-pack.tsx]: Thanks @elenalape, that makes sense! I'll fix this now.

Comment (elenalape) [reply src/pages/github-student-developer-pack.tsx]: Awesome.

**Pull request merged**